pkg/apis/capabilities/v1alpha1/zz_generated.openapi.go

```diff
@@ -1,7 +1,5 @@
 // +build !ignore_autogenerated
 
```

Comment:
This PR does not have changes in the capabilities CRDs so this file shouldn't be commited.
I highly suspect this changes happen because you must be using a different operator-sdk version than 0.8.0 or a more recent Go version than 1.12.x